<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resource/picture/office365.png</normaloff>../resource/picture/office365.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="leftMargin">
        <number>100</number>
       </property>
       <property name="rightMargin">
        <number>130</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../resource/picture/office365.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Office自动化办公助手</string>
         </property>
         <property name="pixmap">
          <pixmap>../resource/picture/office365.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideLeft</enum>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="icon">
         <iconset>
          <normaloff>../resource/picture/powerpoint1.png</normaloff>../resource/picture/powerpoint1.png</iconset>
        </attribute>
        <attribute name="title">
         <string>PPT处理</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>PPT转PDF:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pathPPT2PDF">
                  <property name="cursor">
                   <cursorShape>IBeamCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>D:\Desktop\功能测试\ppt文件</string>
                  </property>
                  <property name="dragEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="chooseButtonPPT2PDF">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string> 选择路径</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="convertButtonPPT2PDF">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>开始转换</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>PPT转JPG:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pathPPT2JPG">
                  <property name="cursor">
                   <cursorShape>IBeamCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>D:\Desktop\功能测试\ppt文件</string>
                  </property>
                  <property name="dragEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="chooseButtonPPT2JPG">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>选择路径</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="convertButtonPPT2JPG">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>开始转换</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>提取图片:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pathImageExtract"/>
                </item>
                <item>
                 <widget class="QPushButton" name="chooseButtonImageExtract">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>选择文件</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="ExtractButton">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string> 开始提取</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="icon">
         <iconset>
          <normaloff>../resource/picture/word1.png</normaloff>../resource/picture/word1.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Word处理</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="labelWord2Pdf">
                  <property name="text">
                   <string>Word转PDF：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pathWord2PDF">
                  <property name="cursor">
                   <cursorShape>IBeamCursor</cursorShape>
                  </property>
                  <property name="acceptDrops">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>D:\Desktop\功能测试\word文件</string>
                  </property>
                  <property name="dragEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="chooseButtonWord2PDF">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string> 选择路径</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="convertButtonWord2PDF">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>开始转换</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>关键字替换:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pathKeywordReplace">
                  <property name="text">
                   <string>D:\Desktop\功能测试\word文件\替换关键字</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="chooseButtonKeywordReplace">
                  <property name="text">
                   <string>选择路径</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="adjustButtonKeyword">
                  <property name="text">
                   <string> 调整参数</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../resource/picture/setting.png</normaloff>../resource/picture/setting.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="icon">
         <iconset>
          <normaloff>../resource/picture/pdf1.png</normaloff>../resource/picture/pdf1.png</iconset>
        </attribute>
        <attribute name="title">
         <string>PDF 处理</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>添加水印:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pathAddWatermark">
               <property name="text">
                <string>D:\Desktop\功能测试\PDF文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="choosePathAddWatermarkButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>选择路径</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="addWatermarkButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string> 参数设置</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resource/picture/setting.png</normaloff>../resource/picture/setting.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>批量加密:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pathEncryption">
               <property name="text">
                <string>D:\Desktop\功能测试\PDF文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="choosePathEncryption">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>选择路径</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="encryptionButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>开始</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>批量解密:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pathDecrypt">
               <property name="text">
                <string>D:\Desktop\功能测试\PDF文件</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="choosePathDecryption">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>选择路径</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="DecryptionButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>开始</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="icon">
         <iconset>
          <normaloff>../resource/picture/file1.png</normaloff>../resource/picture/file1.png</iconset>
        </attribute>
        <attribute name="title">
         <string> 文件处理</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>文件分类：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pathFileClassication">
             <property name="text">
              <string>D:\Desktop\功能测试\文件分类功能展示</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chooseButtonFileClassification">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>选择路径</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>E:/图吧工具箱/skin/user/系统信息.png</normaloff>E:/图吧工具箱/skin/user/系统信息.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="classificationButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>开始处理</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>冗余文件处理：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pathDuplicateFile">
             <property name="text">
              <string>D:\Desktop\功能测试\检测重复文件功能展示</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chooseDuplicateFileButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>选择路径</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../resource/picture/path.png</normaloff>../resource/picture/path.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="duplicateFileButton">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>开始处理</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../resource/picture/start.png</normaloff>../resource/picture/start.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>531</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
